minos/cli/api/init.py

```diff
@@ -9,31 +9,31 @@
 )
 from ..templating import (
     MICROSERVICE_INIT,
+    PROJECT_INIT,
     TemplateProcessor,
 )
 
 app = typer.Typer(add_completion=False)
 
 
-@app.command("init")
-def init() -> None:
-    """Initialize a microservice on the current working directory."""
+@app.command("project")
+def init_project() -> None:
+    """Initialize a project on the current working directory."""
 
-    console.print(":wrench: Initializing new Microservice...\n")
-    processor = TemplateProcessor.from_fetcher(MICROSERVICE_INIT, Path.cwd(), defaults={"name": Path.cwd().name})
+    console.print(":wrench: Initializing new Project...\n")
+    processor = TemplateProcessor.from_fetcher(PROJECT_INIT, Path.cwd(), defaults={"project_name": Path.cwd().name})
     processor.render()
 
 
-@app.command("new")
-def new(path: Path) -> None:
-    """Initialize a microservice on the given directory."""
+@app.command("microservice")
+def init_microservice() -> None:
+    """Initialize a microservice on the current working directory."""
 
-    console.print(":wrench: Creating new Microservice...\n")
-    processor = TemplateProcessor.from_fetcher(MICROSERVICE_INIT, path.absolute(), defaults={"name": path.name})
+    console.print(":wrench: Initializing new Microservice...\n")
+    processor = TemplateProcessor.from_fetcher(MICROSERVICE_INIT, Path.cwd(), defaults={"name": Path.cwd().name})
     processor.render()
 
 
 @app.callback()
 def callback():
-    """Minos microservice CLI."""
-    console.print(":gift: Microservice Utilities :gift:\n")
```

Comment:
I would not remove the `console.print` calls, as they enhances the console messages.

Reply:
IMO the `console.print()` doesn't add anything to the CLI messages since the template itself documents what it's doing.
I've modified the comments to enhance the `minos --help` documentation, though.